pkg/util/bandwidth/linux.go

```diff
@@ -156,12 +157,17 @@ func (t *tcShaper) findCIDRClass(cidr string) (class, handle string, found bool,
 			// expected tc line:
 			// filter parent 1: protocol ip pref 1 u32 fh 800::800 order 2048 key ht 800 bkt 0 flowid 1:1
 			if len(parts) != 19 {
-				return "", "", false, fmt.Errorf("unexpected output from tc: %s %d (%v)", filter, len(parts), parts)
+				return [][]string{}, false, fmt.Errorf("unexpected output from tc: %s %d (%v)", filter, len(parts), parts)
+			} else {
+				resultTmp := []string{parts[18], parts[9]}
+				result = append(result, resultTmp)
 			}
-			return parts[18], parts[9], true, nil
 		}
 	}
-	return "", "", false, nil
+	if len(result) > 0 {
+		return result, true, nil
+	}
+	return [][]string{}, false, nil
```

Comment:
Same as above. Return `result` is enough.

Reply:
It's been modified as you said.